app/components/UserNavigation/index.jsx

```diff
@@ -171,7 +172,7 @@ const UserNavigation = ({
             id={`redireact-to-edit-${userId}-user-page-2`}
             data-user={userId}
           >
-            <img src={pencilIcon} alt="icon" />
+            <Icon icon={pencilIcon} width="18" />
```

Comment:
I see width=18 repeating several times. Maybe create a component for icon of this width?

Reply:
Or maybe even several components for icons of different sizes.